### PR TITLE
Tutorial/utility to easily view assets in habitat sim engine.

### PR DESCRIPTION
**Pull request by jturner65**
## Motivation and Context
This PR adds a simple colab that enables folks to load and view assets into simulator easily.  This will be particularly helpful in testing dataset assets such ReplicaCAD, that perhaps are being iterated upon in an environment that does not have sim installed.

## How Has This Been Tested
C++ and python tests pass locally

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: examples/tutorials/nb_python/asset_viewer.py [hunk 1]
```diff
@@ -0,0 +1,972 @@
...
+# @markdown (double click to show code)
+
+# @markdown This cell defines a number of utility functions used throughout the tutorial to make simulator reconstruction easy:
+# @markdown - make_cfg
+# @markdown - make_default_settings
+# @markdown - make_simulator_from_settings
+
+
+def make_cfg(settings):
+    sim_cfg = habitat_sim.SimulatorConfiguration()
```
Comment (Skylion007) [inline examples/tutorials/nb_python/asset_viewer.py:403]: Why can't we use the utility we made for this?

File: examples/tutorials/nb_python/asset_viewer.py [hunk 2]
```diff
@@ -0,0 +1,972 @@
...
+    obj.rotation = orientation
+
+
+# %%
+# @title Define Visualization Utility Function { display-mode: "form" }
+# @markdown (double click to show code)
+# @markdown - display_sample
+
+# Change to do something like this maybe: https://stackoverflow.com/a/41432704
+def display_sample(
```
Comment (Skylion007) [inline examples/tutorials/nb_python/asset_viewer.py:660]: We really need to modernize and fix this util at some point... At the very least standardize it somewher eso it's easy to refactor later.

File: examples/tutorials/nb_python/asset_viewer.py [hunk 3]
```diff
@@ -0,0 +1,972 @@
...
+    parser.add_argument("--no-make-video", dest="make_video", action="store_false")
+    parser.set_defaults(show_video=True, make_video=True)
+    args, _ = parser.parse_known_args()
+    show_video = args.display
+    display = args.display
+    make_video = args.make_video
+else:
+    show_video = False
+    make_video = False
+    display = False
```
Comment (Skylion007) [inline examples/tutorials/nb_python/asset_viewer.py:710]: All the remaining code would still run though... This kinda defeats the point of `__main__`.

File: examples/tutorials/nb_python/asset_viewer.py [hunk 4]
```diff
@@ -0,0 +1,972 @@
...
+
+    HAS_WIDGETS = True
+except ImportError:
+    HAS_WIDGETS = False
+
+
+if "google.colab" in sys.modules:
+    os.environ["IMAGEIO_FFMPEG_EXE"] = "/usr/bin/ffmpeg"
+
+repo = git.Repo(".", search_parent_directories=True)
```
Comment (Skylion007) [inline examples/tutorials/nb_python/asset_viewer.py:74]: We shoudl lift this into a utility too.

File: examples/tutorials/nb_python/asset_viewer.py [hunk 5]
```diff
@@ -0,0 +1,972 @@
...
+
+# @markdown - simulate
+# @markdown - init_camera_track_config
+# @markdown - restore_camera_track_config
+# @markdown - camera_track_simulate
+
+
+def simulate(sim, dt=1.0, get_frames=True):
+    # simulate dt seconds at 60Hz to the nearest fixed timestep
+    print("Simulating " + str(dt) + " world seconds.")
```
Comment (Skylion007) [inline examples/tutorials/nb_python/asset_viewer.py:565]: Use fstring

File: examples/tutorials/nb_python/asset_viewer.py (hunk 1 above)
Comment (jturner65) [reply examples/tutorials/nb_python/asset_viewer.py:403]: I cannibalized an existing ECCV colab to get started here.  These all need to be updated.

**Comment by Skylion007** [comment]
You should consider adding a smoke_test to test_examples as well.

**Comment by jturner65** [comment]
Some example output : 

https://user-images.githubusercontent.com/1093953/122284800-f46e2f00-cebb-11eb-9bb6-5b784e5c18bf.mp4


https://user-images.githubusercontent.com/1093953/122284818-f932e300-cebb-11eb-8c84-1e7c73181671.mp4

**Review by aclegg3 — APPROVED** [review]
LGTM. Cool utility.

**Pull request merged**